src/shared/utils.ts

```diff
@@ -73,4 +78,27 @@ export const getScheduledSurfaceByGuid = (
     (surface: ScheduledSurface) => surface.guid === guid
   );
 };
+
+export const getCorpusItemFromApprovedItem = (
+  approvedItem: ApprovedItem
+): CorpusItem => {
+  return {
+    id: approvedItem.externalId,
+    url: approvedItem.url,
+    title: approvedItem.title,
+    excerpt: approvedItem.excerpt,
+    authors: approvedItem.authors as ApprovedItemAuthor[],
```

Comment:
Note: we type cast this, because `authors` is optional for `ApprovedItem`, but required for `CorpusItem`. It's probably also required on `scheduledItem.approvedItem`?

Reply:
Good inheritance question...

Reply:
`authors` was made optional because most corpus items don't have one yet - this is a change that was brought in to enable scheduling of Pocket Hits correctly - authors are only used there at the moment. We're working on backfilling authors data for all curated items. Once that is done, we'll be able to make changes both to the admin schema and the underlying database schema.

Reply:
as is, because not all approved items have authors yet, i think this will error out depending on the database record returned. are we okay with that? or do we want to make [`authors` on `CorpusItem`](https://github.com/Pocket/curated-corpus-api/blob/main/src/database/types.ts#L158) optional until the backfill completes?